bin/fin

```diff
@@ -3582,28 +3582,33 @@ run_cli ()
 # --db-user="admin" to override mysql username
 # --db-password="otherpass" to override mysql password
 # --db="drupal" to override database (only applies to queries)
-mysql ()
+_mysql ()
 {
 	check_winpty_found
+	check_docksal_environment
 	eval $(parse_params "$@")
 
+	container_id="${COMPOSE_PROJECT_NAME_SAFE}_db_1"
+	# Bring in MYSQL env variables from the container, so that we can use them here
+	container_mysql_vars=$(docker exec "$container_id" env | grep MYSQL)
+	eval ${container_mysql_vars}
```

Comment:
I think we should separate out these 4 lines into a function named `_mysql_env` and use it in all mysql functions